src/views/docs-view/utils/get-deploy-preview-loader.ts

```diff
@@ -78,6 +78,9 @@ export function getDeployPreviewLoader({
 			if (
 				currentRootDocsPath.productSlugForLoader?.match(
 					/^(terraform|ptfe-releases)/i
+				) &&
+				!__config.flags?.unified_docs_migrated_repos?.find(
```

Comment:
Added to prevent `remark-rewrite-assets` from running twice for terraform product .